### ISSUE TITLE
LP config file not showing up

### ISSUE THREAD

**Issue by PiwosElwy** [lucko/LuckPerms#2121]
I put the LuckPerms.jar file in my plugins folder, I Started the server and stopped it, but the file is still not there.

It's also not showing up in game whenever I type /plugins 
So I have a feeling it's not activating or something

Does anyone know what I should do?

**Comment by BrainStone** [comment]
What server platform/software, MC version and LP file are you using?

**Comment by PiwosElwy** [comment]
I'm using the hosting service "ggservers.com"
Minecraft version is "1.15.2"
and the file I'm using is called "LuckPerms-Bukkit-5.0.72.jar"

I am completely new to all of this so sorry if I'm unclear anywhere, if I am, please correct me.

**Comment by PiwosElwy** [comment]
I'm also running Paperspigot

**Comment by BrainStone** [comment]
Check for any errors during startup.

**Comment by PiwosElwy** [comment]
I found only this 

[Server] ERROR Could not load 'plugins/LuckPerms-Bukkit-5.0.72.jar' in folder 'plugins'

**Comment by BrainStone** [comment]
What are the messages before and after?

**Comment by PiwosElwy** [comment]
Do you want me to post everything that I see in the console? 
Or just a few lines above and under the error message?

**Comment by BrainStone** [comment]
If you're not sure, upload your log. The part until it says ``Done (...)`` (the ``...`` being the time it took to start) is the one that matters. We don't need anything after that for now

**Comment by PiwosElwy** [comment]
Okay, here you go!

30.03 22:33:02 [Multicraft] Received start command
30.03 22:33:02 [Multicraft] Loading server properties
30.03 22:33:02 [Multicraft] Starting server!
30.03 22:33:02 [Multicraft] Loaded config for "PaperSpigot (latest stable)"
30.03 22:33:02 [Multicraft] Updating eula.txt file
30.03 22:33:03 [Server] INFO System Info: Java 1.8 (OpenJDK 64-Bit Server VM 25.242-b08) Host: Linux 4.19-ovh-xxxx-std-ipv6-64 (amd64)
30.03 22:33:03 [Server] INFO Loading libraries, please wait...
30.03 22:33:06 [Server] INFO 2020-03-30 22:33:06,003 main WARN Advanced terminal features are not available in this environment
30.03 22:33:15 [Server] INFO Starting minecraft server version 1.15.2
30.03 22:33:15 [Server] INFO Loading properties
30.03 22:33:16 [Server] INFO This server is running Paper version git-Paper-145 (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)
30.03 22:33:16 [Server] INFO Server Ping Player Sample Count: 12
30.03 22:33:16 [Server] INFO Using 4 threads for Netty based IO
30.03 22:33:16 [Server] INFO Debug logging is disabled
30.03 22:33:16 [Server] INFO Default game type: SURVIVAL
30.03 22:33:16 [Server] INFO Generating keypair
30.03 22:33:17 [Server] INFO Starting Minecraft server on 51.178.89.130:25654
30.03 22:33:17 [Server] INFO Using epoll channel type
30.03 22:33:18 [Server] ERROR Could not load 'plugins/LuckPerms-Bukkit-5.0.72.jar' in folder 'plugins'
30.03 22:33:18 [Server] INFO org.bukkit.plugin.InvalidDescriptionException: Invalid plugin.yml
30.03 22:33:18 [Server] INFO at org.bukkit.plugin.java.JavaPluginLoader.getPluginDescription(JavaPluginLoader.java:167) ~[paperspigot.jar:git-Paper-145]
30.03 22:33:18 [Server] INFO at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:148) ~[paperspigot.jar:git-Paper-145]
30.03 22:33:18 [Server] INFO at org.bukkit.craftbukkit.v1_15_R1.CraftServer.loadPlugins(CraftServer.java:358) ~[paperspigot.jar:git-Paper-145]
30.03 22:33:18 [Server] INFO at net.minecraft.server.v1_15_R1.DedicatedServer.init(DedicatedServer.java:234) ~[paperspigot.jar:git-Paper-145]
30.03 22:33:18 [Server] INFO at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:885) ~[paperspigot.jar:git-Paper-145]
30.03 22:33:18 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_242]
30.03 22:33:18 [Server] INFO Caused by: java.util.zip.ZipException: error in opening zip file
30.03 22:33:18 [Server] INFO at java.util.zip.ZipFile.open(Native Method) ~[?:1.8.0_242]
30.03 22:33:18 [Server] INFO at java.util.zip.ZipFile.<init>(ZipFile.java:225) ~[?:1.8.0_242]
30.03 22:33:18 [Server] INFO at java.util.zip.ZipFile.<init>(ZipFile.java:155) ~[?:1.8.0_242]
30.03 22:33:18 [Server] INFO at java.util.jar.JarFile.<init>(JarFile.java:166) ~[?:1.8.0_242]
30.03 22:33:18 [Server] INFO at java.util.jar.JarFile.<init>(JarFile.java:130) ~[?:1.8.0_242]
30.03 22:33:18 [Server] INFO at org.bukkit.plugin.java.JavaPluginLoader.getPluginDescription(JavaPluginLoader.java:155) ~[paperspigot.jar:git-Paper-145]
30.03 22:33:18 [Server] INFO ... 5 more
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Injector platform set to: InjectorBukkit
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Injecting NMS classes...
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Injecting WorldEdit classes...
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.util.eventbus.EventBus
30.03 22:33:20 [Server] WARN [WorldEdit] Loaded class org.primesoft.asyncworldedit.injector.injected.util.eventbus.IDispatchableEventBus from AsyncWorldEdit v3.6.19 which is not a depend, softdepend or loadbefore of this plugin.
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.math.BlockVector2
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.math.BlockVector3
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.math.Vector2
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.math.Vector3
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.world.block.BlockStateHolder
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.world.block.BaseBlock
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.world.block.BlockState
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.entity.BaseEntity
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.util.Location
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.world.biome.BiomeType
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.world.weather.WeatherType
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.EditSession
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.function.operation.Operations
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.extent.clipboard.BlockArrayClipboard
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.extension.platform.PlayerProxy
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.util.collection.LocatedBlockList
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.extent.reorder.MultiStageReorder
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.extent.reorder.ChunkBatchingExtent
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.regions.AbstractRegion
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.regions.EllipsoidRegion
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.regions.ConvexPolyhedralRegion
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.regions.TransformRegion
30.03 22:33:20 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.regions.RegionIntersection
30.03 22:33:21 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.regions.NullRegion
30.03 22:33:21 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.regions.Polygonal2DRegion
30.03 22:33:21 [Multicraft] Skipped 7 lines due to rate limit (30/s)
30.03 22:33:21 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.bukkit.BukkitEntity
30.03 22:33:21 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.command.RegionCommandsRegistration
30.03 22:33:21 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.command.tool.BrushTool
30.03 22:33:21 [Server] INFO [AsyncWorldEdit] Modify class com.sk89q.worldedit.command.tool.BlockReplacer
30.03 22:33:21 [Server] INFO [AsyncWorldEdit] Creating class org/primesoft/asyncworldedit/worldedit/entity/PlayerWrapper
30.03 22:33:21 [Server] INFO [AsyncWorldEdit] Creating class org/primesoft/asyncworldedit/worldedit/entity/NoPermsPlayer
30.03 22:33:21 [Server] INFO [AsyncWorldEdit] Creating class org/primesoft/asyncworldedit/worldedit/extension/platform/NoPermsActor
30.03 22:33:21 [Server] INFO [AsyncWorldEdit] Creating class org/primesoft/asyncworldedit/injector/injected/entity/IPlayerFactory
30.03 22:33:21 [Server] INFO [AsyncWorldEdit] Creating class org/primesoft/asyncworldedit/injector/injected/extension/platform/IActorFactory
30.03 22:33:21 [Server] INFO [AsyncWorldEdit] Injector Enabled
30.03 22:33:21 [Server] INFO [AsyncWorldEdit] New class factory set to: org.primesoft.asyncworldedit.asyncinjector.async.AsyncClassFactory
30.03 22:33:22 [Server] INFO [WorldEdit] Loading WorldEdit v7.1.0;8e55131
30.03 22:33:23 [Server] INFO Got request to register class com.sk89q.worldedit.bukkit.BukkitServerInterface with WorldEdit [com.sk89q.worldedit.extension.platform.PlatformManager@5b41493d]
30.03 22:33:23 [Server] INFO [AsyncWorldEdit] Loading AsyncWorldEdit v3.6.19
30.03 22:33:23 [Server] INFO [AsyncWorldEdit] Plugin installed
30.03 22:33:23 [Server] INFO [mcMMO] Loading mcMMO v2.1.124
30.03 22:33:23 [Server] INFO [Vault] Loading Vault v1.7.2-b107
30.03 22:33:23 [Server] INFO [ChestShop] Loading ChestShop v3.10.1 (build 174)
30.03 22:33:23 [Server] INFO [Essentials] Loading Essentials v2.17.2.0
30.03 22:33:23 [Server] INFO [EssentialsSpawn] Loading EssentialsSpawn v2.17.2.0
30.03 22:33:23 [Server] INFO [EssentialsChat] Loading EssentialsChat v2.17.2.0
30.03 22:33:23 [Server] INFO Server permissions file permissions.yml is empty, ignoring it
30.03 22:33:23 [Server] INFO [WorldEdit] Enabling WorldEdit v7.1.0;8e55131
30.03 22:33:23 [Server] WARN [WorldEdit] Loaded class net.milkbowl.vault.permission.Permission from Vault v1.7.2-b107 which is not a depend, softdepend or loadbefore of this plugin.
30.03 22:33:23 [Server] INFO WEPIF: Using the Bukkit Permissions API.
30.03 22:33:25 [Server] INFO Using com.sk89q.worldedit.bukkit.adapter.impl.Spigot_v1_15_R2 as the Bukkit adapter
30.03 22:33:27 [Server] INFO [AsyncWorldEdit] Enabling AsyncWorldEdit v3.6.19
30.03 22:33:27 [Server] INFO [AsyncWorldEdit] Platform set to Bukkit API
30.03 22:33:27 [Server] INFO [AsyncWorldEdit] ===================================================================================
30.03 22:33:27 [Server] INFO [AsyncWorldEdit] =     ___                         _       __           __    ________    ___ __   =
30.03 22:33:27 [Server] INFO [AsyncWorldEdit] =    /   |  _______  ______  ____| |     / /___  _____/ /___/ / ____/___/ (_) /_  =
30.03 22:33:27 [Server] INFO [AsyncWorldEdit] =   / /| | / ___/ / / / __ \/ ___/ | /| / / __ \/ ___/ / __  / __/ / __  / / __/  =
30.03 22:33:27 [Server] INFO [AsyncWorldEdit] =  / ___ |(__  ) /_/ / / / / /__ | |/ |/ / /_/ / /  / / /_/ / /___/ /_/ / / /_    =
30.03 22:33:27 [Server] INFO [AsyncWorldEdit] = /_/  |_/____/\__, /_/ /_/\___/ |__/|__/\____/_/  /_/\__,_/_____/\__,_/_/\__/    =
30.03 22:33:27 [Server] INFO [AsyncWorldEdit] =             /____/                                                              =
30.03 22:33:27 [Server] INFO [AsyncWorldEdit] = Author: SBPrime                                                                 =
30.03 22:33:27 [Server] INFO [AsyncWorldEdit] =                                ________________                                 =
30.03 22:33:27 [Server] INFO [AsyncWorldEdit] =                            ___/  free version  \___                             =
30.03 22:33:27 [Server] INFO [AsyncWorldEdit] ===================================================================================
30.03 22:33:37 [Server] INFO [AsyncWorldEdit] Initializing WorldEdit.
30.03 22:33:37 [Server] INFO [AsyncWorldEdit] Undo cleanup started...
30.03 22:33:37 [Server] INFO [AsyncWorldEdit] ...undo cleanup done.
30.03 22:33:37 [Server] INFO [AsyncWorldEdit] No matching adapter found for Bukkit.v1_15_R1, DirectChunkAPI disabled
30.03 22:33:37 [Server] INFO [AsyncWorldEdit] PlotMeFix set to class org.primesoft.asyncworldedit.plotme.NullFix.
30.03 22:33:37 [Server] INFO [AsyncWorldEdit] Enabled
30.03 22:33:37 [Server] INFO [Vault] Enabling Vault v1.7.2-b107
30.03 22:33:38 [Server] WARN [Vault] Loaded class com.earth2me.essentials.api.Economy from Essentials v2.17.2.0 which is not a depend, softdepend or loadbefore of this plugin.
30.03 22:33:38 [Server] INFO [Vault] [Economy] Essentials Economy found: Waiting
30.03 22:33:38 [Server] INFO [Vault] [Permission] SuperPermissions loaded as backup permission system.
30.03 22:33:38 [Server] INFO [Vault] Enabled Version 1.7.2-b107
30.03 22:33:38 [Server] INFO WEPIF: Vault detected! Using Vault for permissions
30.03 22:33:38 [Server] INFO Preparing level "world"
30.03 22:33:38 [Server] INFO Reloading ResourceManager: Default, bukkit
30.03 22:33:39 [Server] INFO [AsyncWorldEdit] You have an old version of the plugin. Your version: 3.6.19, available version: 3.6.20
30.03 22:33:39 [Server] INFO Loaded 6 recipes
30.03 22:33:40 [Server] INFO -------- World Settings For [world] --------
30.03 22:33:40 [Server] INFO Mob Spawn Range: 8
30.03 22:33:40 [Server] INFO Custom Map Seeds:  Village: 10387312 Desert: 14357617 Igloo: 14357618 Jungle: 14357619 Swamp: 14357620 Monument: 10387313 Ocean: 14357621 Shipwreck: 165745295 Slime: 987234911
30.03 22:33:40 [Server] INFO Max TNT Explosions: 100
30.03 22:33:40 [Server] INFO Tile Max Tick Time: 50ms Entity max Tick Time: 50ms
30.03 22:33:40 [Server] INFO Cactus Growth Modifier: 100%
30.03 22:33:40 [Server] INFO Cane Growth Modifier: 100%
30.03 22:33:40 [Server] INFO Melon Growth Modifier: 100%
30.03 22:33:40 [Server] INFO Mushroom Growth Modifier: 100%
30.03 22:33:40 [Server] INFO Pumpkin Growth Modifier: 100%
30.03 22:33:40 [Server] INFO Sapling Growth Modifier: 100%
30.03 22:33:40 [Server] INFO Beetroot Growth Modifier: 100%
30.03 22:33:40 [Server] INFO Carrot Growth Modifier: 100%
30.03 22:33:40 [Server] INFO Potato Growth Modifier: 100%
30.03 22:33:40 [Server] INFO Wheat Growth Modifier: 100%
30.03 22:33:40 [Server] INFO NetherWart Growth Modifier: 100%
30.03 22:33:40 [Server] INFO Vine Growth Modifier: 100%
30.03 22:33:40 [Server] INFO Cocoa Growth Modifier: 100%
30.03 22:33:40 [Server] INFO Bamboo Growth Modifier: 100%
30.03 22:33:40 [Server] INFO SweetBerry Growth Modifier: 100%
30.03 22:33:40 [Server] INFO Kelp Growth Modifier: 100%
30.03 22:33:40 [Server] INFO Entity Activation Range: An 32 / Mo 32 / Ra 48 / Mi 16 / Tiv true
30.03 22:33:40 [Server] INFO Entity Tracking Range: Pl 48 / An 48 / Mo 48 / Mi 32 / Other 64
30.03 22:33:40 [Server] INFO Hopper Transfer: 8 Hopper Check: 1 Hopper Amount: 1
30.03 22:33:40 [Server] INFO Experience Merge Radius: 3.0
30.03 22:33:40 [Server] INFO View Distance: 10
30.03 22:33:40 [Server] INFO Allow Zombie Pigmen to spawn from portal blocks: true
30.03 22:33:40 [Server] INFO Item Despawn Rate: 6000
30.03 22:33:40 [Server] INFO Item Merge Radius: 2.5
30.03 22:33:40 [Server] INFO Arrow Despawn Rate: 1200 Trident Respawn Rate:1200
30.03 22:33:42 [Multicraft] Skipped 68 lines due to rate limit (30/s)
30.03 22:33:42 [Server] INFO Preparing spawn area: 0%
30.03 22:33:42 [Server] INFO Preparing spawn area: 0%
30.03 22:33:42 [Server] INFO Preparing spawn area: 0%
30.03 22:33:43 [Server] INFO Preparing spawn area: 0%
30.03 22:33:43 [Server] INFO Preparing spawn area: 0%
30.03 22:33:44 [Server] INFO Preparing spawn area: 100%
30.03 22:33:44 [Server] INFO Preparing spawn area: 100%
30.03 22:33:45 [Server] INFO Loaded 441 spawn chunks for world world
30.03 22:33:45 [Server] INFO Time elapsed: 3453 ms
30.03 22:33:45 [Server] INFO Preparing start region for dimension 'world_nether'/minecraft:the_nether
30.03 22:33:45 [Server] INFO Preparing spawn area: 0%
30.03 22:33:45 [Server] INFO Preparing spawn area: 61%
30.03 22:33:46 [Server] INFO Preparing spawn area: 95%
30.03 22:33:46 [Server] INFO Preparing spawn area: 100%
30.03 22:33:47 [Server] INFO Preparing spawn area: 100%
30.03 22:33:47 [Server] INFO Preparing spawn area: 100%
30.03 22:33:48 [Server] INFO Preparing spawn area: 100%
30.03 22:33:48 [Server] INFO Preparing spawn area: 100%
30.03 22:33:49 [Server] INFO Preparing spawn area: 100%
30.03 22:33:49 [Server] INFO Preparing spawn area: 100%
30.03 22:33:49 [Server] INFO Loaded 441 spawn chunks for world world_nether
30.03 22:33:49 [Server] INFO Time elapsed: 4759 ms
30.03 22:33:49 [Server] INFO Preparing start region for dimension 'world_the_end'/minecraft:the_end
30.03 22:33:50 [Server] INFO Preparing spawn area: 0%
30.03 22:33:50 [Server] INFO Preparing spawn area: 100%
30.03 22:33:50 [Server] INFO Preparing spawn area: 100%
30.03 22:33:51 [Server] INFO Preparing spawn area: 100%
30.03 22:33:51 [Server] INFO Loaded 441 spawn chunks for world world_the_end
30.03 22:33:51 [Server] INFO Time elapsed: 1509 ms
30.03 22:33:51 [Server] INFO [mcMMO] Enabling mcMMO v2.1.124
30.03 22:33:51 [Server] INFO [mcMMO] No support for repair item NETHERITE_SWORD in this version of Minecraft, skipping.
30.03 22:33:51 [Server] INFO [mcMMO] No support for repair item NETHERITE_SHOVEL in this version of Minecraft, skipping.
30.03 22:33:51 [Server] INFO [mcMMO] No support for repair item NETHERITE_PICKAXE in this version of Minecraft, skipping.
30.03 22:33:51 [Server] INFO [mcMMO] No support for repair item NETHERITE_AXE in this version of Minecraft, skipping.
30.03 22:33:51 [Server] INFO [mcMMO] No support for repair item NETHERITE_HOE in this version of Minecraft, skipping.
30.03 22:33:51 [Server] INFO [mcMMO] No support for repair item NETHERITE_HELMET in this version of Minecraft, skipping.
30.03 22:33:51 [Server] INFO [mcMMO] No support for repair item NETHERITE_CHESTPLATE in this version of Minecraft, skipping.
30.03 22:33:51 [Server] INFO [mcMMO] No support for repair item NETHERITE_LEGGINGS in this version of Minecraft, skipping.
30.03 22:33:51 [Server] INFO [mcMMO] No support for repair item NETHERITE_BOOTS in this version of Minecraft, skipping.
30.03 22:33:52 [Server] INFO [mcMMO] No support for salvage item NETHERITE_SWORD in this version of Minecraft, skipping.
30.03 22:33:52 [Server] INFO [mcMMO] No support for salvage item NETHERITE_SHOVEL in this version of Minecraft, skipping.
30.03 22:33:52 [Server] INFO [mcMMO] No support for salvage item NETHERITE_PICKAXE in this version of Minecraft, skipping.
30.03 22:33:52 [Server] INFO [mcMMO] No support for salvage item NETHERITE_AXE in this version of Minecraft, skipping.
30.03 22:33:52 [Server] INFO [mcMMO] No support for salvage item NETHERITE_HOE in this version of Minecraft, skipping.
30.03 22:33:52 [Server] INFO [mcMMO] No support for salvage item NETHERITE_HELMET in this version of Minecraft, skipping.
30.03 22:33:52 [Server] INFO [mcMMO] No support for salvage item NETHERITE_CHESTPLATE in this version of Minecraft, skipping.
30.03 22:33:52 [Server] INFO [mcMMO] No support for salvage item NETHERITE_LEGGINGS in this version of Minecraft, skipping.
30.03 22:33:52 [Server] INFO [mcMMO] No support for salvage item NETHERITE_BOOTS in this version of Minecraft, skipping.
30.03 22:33:52 [Server] INFO [mcMMO] enabling Acrobatics Skills
30.03 22:33:52 [Server] INFO [mcMMO] registered subskill: Roll
30.03 22:33:52 [Server] INFO [mcMMO] 0 entries in mcMMO World Blacklist
30.03 22:33:52 [Server] INFO [ChestShop] Enabling ChestShop v3.10.1 (build 174)
30.03 22:33:53 [Server] INFO [ChestShop] Using Essentials Economy as the Economy provider now.
30.03 22:33:53 [Server] INFO [ChestShop] Vault loaded!
30.03 22:33:53 [Server] INFO [ChestShop] Using Paper's BlockDestroyEvent instead of the BlockPhysicsEvent!
30.03 22:33:53 [Server] INFO [Essentials] Enabling Essentials v2.17.2.0
30.03 22:33:53 [Server] INFO [Essentials] Attempting to convert old kits in config.yml to new kits.yml
30.03 22:33:53 [Server] INFO [Essentials] No kits found to migrate.
30.03 22:33:53 [Server] INFO [Essentials] Using 1.8.3+ BlockStateMeta provider as mob spawner provider.
30.03 22:33:53 [Server] INFO [Essentials] Using 1.13+ flat spawn egg provider as spawn egg provider.
30.03 22:33:53 [Server] INFO [Essentials] Using 1.9+ BasePotionData provider as potion meta provider.
30.03 22:33:54 [Server] INFO [Essentials] Loaded 21018 items from items.json.
30.03 22:33:54 [Server] INFO [Essentials] Using locale en_US
30.03 22:33:54 [Server] INFO [Essentials] ServerListPingEvent: Spigot iterator API
30.03 22:33:54 [Server] INFO [Essentials] Starting Metrics. Opt-out using the global bStats config.
30.03 22:33:54 [Server] INFO [Vault] [Vault][Economy] Essentials Economy hooked.
30.03 22:33:54 [Server] INFO [Essentials] Using superperms-based permissions.
30.03 22:33:54 [Server] INFO [EssentialsSpawn] Enabling EssentialsSpawn v2.17.2.0
30.03 22:33:54 [Server] INFO [EssentialsChat] Enabling EssentialsChat v2.17.2.0

**Comment by BrainStone** [comment]
Frankly it looks like the jar you downloaded got corrupted. Try downloading it again.

**Comment by PiwosElwy** [comment]
I have downloaded the latest version from luckperms.net 
"LuckPerms-Bukkit-5.0.114.jar"
And it's still giving me the same error message.

**Comment by BrainStone** [comment]
Make sure that when you upload the file over FTP that the transmission mode is either auto or binary.

**Comment by PiwosElwy** [comment]
Alright and how do I do that? 
I just drag and drop the files into the FTP and it automatically puts it into the folder

**Comment by BrainStone** [comment]
Check the (connection) settings.

**Comment by PiwosElwy** [comment]
I'm in auto mode. 
Is that ok? Do in need to change that? If I do, how?

**Comment by BrainStone** [comment]
Should be fine. You can try binary mode.

**Comment by PiwosElwy** [comment]
It doesn't seem that I can. I contact their support team and they just said, that I have to ask the community, which leads me to nowhere since the community is not active at all. 

Anyway I gave up, I just found away to fix the problem that I had without using LuckPerms. 
Thank you so much for helping me even if the issue didn't get resolved the way I hoped it would, the time it took to respond was incredibly short on your end and I really appreciated that, since I have little to no clue what I'm doing.  My issue is now resolved, although I still have no clue why it didn't work, maybe it was just a fluke.
